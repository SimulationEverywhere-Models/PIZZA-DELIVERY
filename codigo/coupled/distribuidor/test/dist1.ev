00:00:01:000 Carga_in_dist 2
00:00:02:000 Carga_in_dist 3
00:00:03:000 Carga_in_dist 4
00:00:04:000 Carga_in_dist 5
00:00:05:000 Carga_in_dist 6
00:00:06:000 Carga_in_dist 7
00:00:07:000 Carga_in_dist 8
00:00:08:000 Carga_in_dist 10
