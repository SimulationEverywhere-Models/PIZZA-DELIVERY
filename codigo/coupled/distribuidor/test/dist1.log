Mensaje I / 00:00:00:000 / Root(00) para top(01)
Mensaje I / 00:00:00:000 / top(01) para sel(02)
Mensaje I / 00:00:00:000 / top(01) para moto1(03)
Mensaje I / 00:00:00:000 / top(01) para moto2(04)
Mensaje I / 00:00:00:000 / top(01) para moto3(05)
Mensaje I / 00:00:00:000 / top(01) para moto4(06)
Mensaje D / 00:00:00:000 / sel(02) / ... para top(01)
Mensaje D / 00:00:00:000 / moto1(03) / ... para top(01)
Mensaje D / 00:00:00:000 / moto2(04) / ... para top(01)
Mensaje D / 00:00:00:000 / moto3(05) / ... para top(01)
Mensaje D / 00:00:00:000 / moto4(06) / ... para top(01)
Mensaje D / 00:00:00:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:01:000 / Root(00) / carga_in_dist /      2.00000 para top(01)
Mensaje X / 00:00:01:000 / top(01) / carga_in /      2.00000 para sel(02)
Mensaje D / 00:00:01:000 / sel(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:01:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:01:000 / Root(00) para top(01)
Mensaje * / 00:00:01:000 / top(01) para sel(02)
Mensaje Y / 00:00:01:000 / sel(02) / sel_out1 /      2.00000 para top(01)
Mensaje D / 00:00:01:000 / sel(02) / ... para top(01)
Mensaje X / 00:00:01:000 / top(01) / carga_mi /      2.00000 para moto1(03)
Mensaje D / 00:00:01:000 / moto1(03) / 00:00:03:000 para top(01)
Mensaje D / 00:00:01:000 / top(01) / 00:00:03:000 para Root(00)
Mensaje X / 00:00:02:000 / Root(00) / carga_in_dist /      3.00000 para top(01)
Mensaje X / 00:00:02:000 / top(01) / carga_in /      3.00000 para sel(02)
Mensaje D / 00:00:02:000 / sel(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:02:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:02:000 / Root(00) para top(01)
Mensaje * / 00:00:02:000 / top(01) para sel(02)
Mensaje Y / 00:00:02:000 / sel(02) / sel_out1 /      3.00000 para top(01)
Mensaje D / 00:00:02:000 / sel(02) / ... para top(01)
Mensaje X / 00:00:02:000 / top(01) / carga_mi /      3.00000 para moto1(03)
Mensaje D / 00:00:02:000 / moto1(03) / 00:00:02:000 para top(01)
Mensaje D / 00:00:02:000 / top(01) / 00:00:02:000 para Root(00)
Mensaje X / 00:00:03:000 / Root(00) / carga_in_dist /      4.00000 para top(01)
Mensaje X / 00:00:03:000 / top(01) / carga_in /      4.00000 para sel(02)
Mensaje D / 00:00:03:000 / sel(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:03:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:03:000 / Root(00) para top(01)
Mensaje * / 00:00:03:000 / top(01) para sel(02)
Mensaje Y / 00:00:03:000 / sel(02) / sel_out1 /      4.00000 para top(01)
Mensaje D / 00:00:03:000 / sel(02) / ... para top(01)
Mensaje X / 00:00:03:000 / top(01) / carga_mi /      4.00000 para moto1(03)
Mensaje D / 00:00:03:000 / moto1(03) / 00:00:00:000 para top(01)
Mensaje D / 00:00:03:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:03:000 / Root(00) para top(01)
Mensaje * / 00:00:03:000 / top(01) para moto1(03)
Mensaje Y / 00:00:03:000 / moto1(03) / ocupada_mi /      1.00000 para top(01)
Mensaje D / 00:00:03:000 / moto1(03) / 00:00:00:150 para top(01)
Mensaje X / 00:00:03:000 / top(01) / esperar_m1 /      1.00000 para sel(02)
Mensaje D / 00:00:03:000 / sel(02) / ... para top(01)
Mensaje D / 00:00:03:000 / top(01) / 00:00:00:150 para Root(00)
Mensaje * / 00:00:03:150 / Root(00) para top(01)
Mensaje * / 00:00:03:150 / top(01) para moto1(03)
Mensaje Y / 00:00:03:150 / moto1(03) / entrega_mi /      2.00000 para top(01)
Mensaje D / 00:00:03:150 / moto1(03) / 00:00:00:090 para top(01)
Mensaje Y / 00:00:03:150 / top(01) / carga_out_dist /      2.00000 para Root(00)
Mensaje D / 00:00:03:150 / top(01) / 00:00:00:090 para Root(00)
Mensaje * / 00:00:03:240 / Root(00) para top(01)
Mensaje * / 00:00:03:240 / top(01) para moto1(03)
Mensaje Y / 00:00:03:240 / moto1(03) / entrega_mi /      3.00000 para top(01)
Mensaje D / 00:00:03:240 / moto1(03) / 00:00:00:060 para top(01)
Mensaje Y / 00:00:03:240 / top(01) / carga_out_dist /      3.00000 para Root(00)
Mensaje D / 00:00:03:240 / top(01) / 00:00:00:060 para Root(00)
Mensaje * / 00:00:03:300 / Root(00) para top(01)
Mensaje * / 00:00:03:300 / top(01) para moto1(03)
Mensaje Y / 00:00:03:300 / moto1(03) / entrega_mi /      4.00000 para top(01)
Mensaje D / 00:00:03:300 / moto1(03) / 00:00:00:300 para top(01)
Mensaje Y / 00:00:03:300 / top(01) / carga_out_dist /      4.00000 para Root(00)
Mensaje D / 00:00:03:300 / top(01) / 00:00:00:300 para Root(00)
Mensaje * / 00:00:03:600 / Root(00) para top(01)
Mensaje * / 00:00:03:600 / top(01) para moto1(03)
Mensaje Y / 00:00:03:600 / moto1(03) / libre_mi /      1.00000 para top(01)
Mensaje D / 00:00:03:600 / moto1(03) / ... para top(01)
Mensaje X / 00:00:03:600 / top(01) / lista_m1 /      1.00000 para sel(02)
Mensaje D / 00:00:03:600 / sel(02) / ... para top(01)
Mensaje D / 00:00:03:600 / top(01) / ... para Root(00)
Mensaje X / 00:00:04:000 / Root(00) / carga_in_dist /      5.00000 para top(01)
Mensaje X / 00:00:04:000 / top(01) / carga_in /      5.00000 para sel(02)
Mensaje D / 00:00:04:000 / sel(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:04:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:04:000 / Root(00) para top(01)
Mensaje * / 00:00:04:000 / top(01) para sel(02)
Mensaje Y / 00:00:04:000 / sel(02) / sel_out1 /      5.00000 para top(01)
Mensaje D / 00:00:04:000 / sel(02) / ... para top(01)
Mensaje X / 00:00:04:000 / top(01) / carga_mi /      5.00000 para moto1(03)
Mensaje D / 00:00:04:000 / moto1(03) / 00:00:03:000 para top(01)
Mensaje D / 00:00:04:000 / top(01) / 00:00:03:000 para Root(00)
Mensaje X / 00:00:05:000 / Root(00) / carga_in_dist /      6.00000 para top(01)
Mensaje X / 00:00:05:000 / top(01) / carga_in /      6.00000 para sel(02)
Mensaje D / 00:00:05:000 / sel(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:05:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:05:000 / Root(00) para top(01)
Mensaje * / 00:00:05:000 / top(01) para sel(02)
Mensaje Y / 00:00:05:000 / sel(02) / sel_out1 /      6.00000 para top(01)
Mensaje D / 00:00:05:000 / sel(02) / ... para top(01)
Mensaje X / 00:00:05:000 / top(01) / carga_mi /      6.00000 para moto1(03)
Mensaje D / 00:00:05:000 / moto1(03) / 00:00:02:000 para top(01)
Mensaje D / 00:00:05:000 / top(01) / 00:00:02:000 para Root(00)
Mensaje X / 00:00:06:000 / Root(00) / carga_in_dist /      7.00000 para top(01)
Mensaje X / 00:00:06:000 / top(01) / carga_in /      7.00000 para sel(02)
Mensaje D / 00:00:06:000 / sel(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:06:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:06:000 / Root(00) para top(01)
Mensaje * / 00:00:06:000 / top(01) para sel(02)
Mensaje Y / 00:00:06:000 / sel(02) / sel_out1 /      7.00000 para top(01)
Mensaje D / 00:00:06:000 / sel(02) / ... para top(01)
Mensaje X / 00:00:06:000 / top(01) / carga_mi /      7.00000 para moto1(03)
Mensaje D / 00:00:06:000 / moto1(03) / 00:00:00:000 para top(01)
Mensaje D / 00:00:06:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:06:000 / Root(00) para top(01)
Mensaje * / 00:00:06:000 / top(01) para moto1(03)
Mensaje Y / 00:00:06:000 / moto1(03) / ocupada_mi /      1.00000 para top(01)
Mensaje D / 00:00:06:000 / moto1(03) / 00:00:00:000 para top(01)
Mensaje X / 00:00:06:000 / top(01) / esperar_m1 /      1.00000 para sel(02)
Mensaje D / 00:00:06:000 / sel(02) / ... para top(01)
Mensaje D / 00:00:06:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:06:000 / Root(00) para top(01)
Mensaje * / 00:00:06:000 / top(01) para moto1(03)
Mensaje Y / 00:00:06:000 / moto1(03) / entrega_mi /      7.00000 para top(01)
Mensaje D / 00:00:06:000 / moto1(03) / 00:00:00:030 para top(01)
Mensaje Y / 00:00:06:000 / top(01) / carga_out_dist /      7.00000 para Root(00)
Mensaje D / 00:00:06:000 / top(01) / 00:00:00:030 para Root(00)
Mensaje * / 00:00:06:030 / Root(00) para top(01)
Mensaje * / 00:00:06:030 / top(01) para moto1(03)
Mensaje Y / 00:00:06:030 / moto1(03) / entrega_mi /      6.00000 para top(01)
Mensaje D / 00:00:06:030 / moto1(03) / 00:00:00:330 para top(01)
Mensaje Y / 00:00:06:030 / top(01) / carga_out_dist /      6.00000 para Root(00)
Mensaje D / 00:00:06:030 / top(01) / 00:00:00:330 para Root(00)
Mensaje * / 00:00:06:360 / Root(00) para top(01)
Mensaje * / 00:00:06:360 / top(01) para moto1(03)
Mensaje Y / 00:00:06:360 / moto1(03) / entrega_mi /      5.00000 para top(01)
Mensaje D / 00:00:06:360 / moto1(03) / 00:00:00:360 para top(01)
Mensaje Y / 00:00:06:360 / top(01) / carga_out_dist /      5.00000 para Root(00)
Mensaje D / 00:00:06:360 / top(01) / 00:00:00:360 para Root(00)
Mensaje * / 00:00:06:720 / Root(00) para top(01)
Mensaje * / 00:00:06:720 / top(01) para moto1(03)
Mensaje Y / 00:00:06:720 / moto1(03) / libre_mi /      1.00000 para top(01)
Mensaje D / 00:00:06:720 / moto1(03) / ... para top(01)
Mensaje X / 00:00:06:720 / top(01) / lista_m1 /      1.00000 para sel(02)
Mensaje D / 00:00:06:720 / sel(02) / ... para top(01)
Mensaje D / 00:00:06:720 / top(01) / ... para Root(00)
Mensaje X / 00:00:07:000 / Root(00) / carga_in_dist /      8.00000 para top(01)
Mensaje X / 00:00:07:000 / top(01) / carga_in /      8.00000 para sel(02)
Mensaje D / 00:00:07:000 / sel(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:07:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:07:000 / Root(00) para top(01)
Mensaje * / 00:00:07:000 / top(01) para sel(02)
Mensaje Y / 00:00:07:000 / sel(02) / sel_out1 /      8.00000 para top(01)
Mensaje D / 00:00:07:000 / sel(02) / ... para top(01)
Mensaje X / 00:00:07:000 / top(01) / carga_mi /      8.00000 para moto1(03)
Mensaje D / 00:00:07:000 / moto1(03) / 00:00:03:000 para top(01)
Mensaje D / 00:00:07:000 / top(01) / 00:00:03:000 para Root(00)
Mensaje X / 00:00:08:000 / Root(00) / carga_in_dist /     10.00000 para top(01)
Mensaje X / 00:00:08:000 / top(01) / carga_in /     10.00000 para sel(02)
Mensaje D / 00:00:08:000 / sel(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:08:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:08:000 / Root(00) para top(01)
Mensaje * / 00:00:08:000 / top(01) para sel(02)
Mensaje Y / 00:00:08:000 / sel(02) / sel_out1 /     10.00000 para top(01)
Mensaje D / 00:00:08:000 / sel(02) / ... para top(01)
Mensaje X / 00:00:08:000 / top(01) / carga_mi /     10.00000 para moto1(03)
Mensaje D / 00:00:08:000 / moto1(03) / 00:00:02:000 para top(01)
Mensaje D / 00:00:08:000 / top(01) / 00:00:02:000 para Root(00)
Mensaje * / 00:00:10:000 / Root(00) para top(01)
Mensaje * / 00:00:10:000 / top(01) para moto1(03)
Mensaje Y / 00:00:10:000 / moto1(03) / ocupada_mi /      1.00000 para top(01)
Mensaje D / 00:00:10:000 / moto1(03) / 00:00:00:030 para top(01)
Mensaje X / 00:00:10:000 / top(01) / esperar_m1 /      1.00000 para sel(02)
Mensaje D / 00:00:10:000 / sel(02) / ... para top(01)
Mensaje D / 00:00:10:000 / top(01) / 00:00:00:030 para Root(00)
Mensaje * / 00:00:10:030 / Root(00) para top(01)
Mensaje * / 00:00:10:030 / top(01) para moto1(03)
Mensaje Y / 00:00:10:030 / moto1(03) / entrega_mi /     10.00000 para top(01)
Mensaje D / 00:00:10:030 / moto1(03) / 00:00:00:090 para top(01)
Mensaje Y / 00:00:10:030 / top(01) / carga_out_dist /     10.00000 para Root(00)
Mensaje D / 00:00:10:030 / top(01) / 00:00:00:090 para Root(00)
Mensaje * / 00:00:10:120 / Root(00) para top(01)
Mensaje * / 00:00:10:120 / top(01) para moto1(03)
Mensaje Y / 00:00:10:120 / moto1(03) / entrega_mi /      8.00000 para top(01)
Mensaje D / 00:00:10:120 / moto1(03) / 00:00:00:120 para top(01)
Mensaje Y / 00:00:10:120 / top(01) / carga_out_dist /      8.00000 para Root(00)
Mensaje D / 00:00:10:120 / top(01) / 00:00:00:120 para Root(00)
Mensaje * / 00:00:10:240 / Root(00) para top(01)
Mensaje * / 00:00:10:240 / top(01) para moto1(03)
Mensaje Y / 00:00:10:240 / moto1(03) / libre_mi /      1.00000 para top(01)
Mensaje D / 00:00:10:240 / moto1(03) / ... para top(01)
Mensaje X / 00:00:10:240 / top(01) / lista_m1 /      1.00000 para sel(02)
Mensaje D / 00:00:10:240 / sel(02) / ... para top(01)
Mensaje D / 00:00:10:240 / top(01) / ... para Root(00)
