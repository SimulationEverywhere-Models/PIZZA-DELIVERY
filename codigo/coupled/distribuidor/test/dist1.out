00:00:03:150 carga_out_dist 2
00:00:03:240 carga_out_dist 3
00:00:03:300 carga_out_dist 4
00:00:06:000 carga_out_dist 7
00:00:06:030 carga_out_dist 6
00:00:06:360 carga_out_dist 5
00:00:10:030 carga_out_dist 10
00:00:10:120 carga_out_dist 8
